src/h5web/explorer/Explorer.module.css

```diff
@@ -45,3 +50,23 @@
   margin-left: -0.375rem;
   margin-right: 0.125rem;
 }
+
+.spinner {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.75rem;
```

Comment:
This is the bit that's a little hacky: the spinner is the fallback of a `Suspense` element, which is rendered _after_ a domain/group button. It needs to be taken off the flow of the page and _moved_ above the button.

Reply:
When React's Concurrent mode finally comes out, we'll be able to use `useTransition` instead of `Suspense` as explained here: https://reactjs.org/docs/concurrent-mode-patterns.html#transitions